edge/pkg/edged/status/status_manager.go

```diff
@@ -73,6 +74,8 @@ func (m *manager) updatePodStatus() {
 			}
 			continue
 		}
+		m.mu.Lock()
+		defer m.mu.Unlock()
```

Comment:
not correct, in range loop using the one lock, it will cause deadlock.

Reply:
oh, sorry~